.vscode/scripts/watch-webapp.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
I can't run these shell script tasks on Windows :( If you could write those shell scripts as npm commands, that would be much more platform independent.